UAA/component.yaml

```diff
@@ -17,305 +18,241 @@ references:
   path: https://github.com/cloudfoundry/cloud_controller_ng/blob/master/spec/unit/access/space_access_spec.rb
   type: URL
 satisfies:
-- control_key: AC-10
+- control_key: AC-2
   covered_by: []
   implementation_status: none
-  narrative: 'Cloud.gov does not support capability to limit active sessions and as
-    a cloud-based system it was not designed to limit the number of active sessions.
-
-    '
-  standard_key: NIST-800-53
-- control_key: IA-2 (5)
-  covered_by: []
-  implementation_status: none
-  narrative: 'This controls is not applicable. here are are no group accounts within
-    the Cloud.Gov platform.
-
-    '
+  narrative:
+  - key: j
+    text: User accounts will be monitored monthly and accounts
+      will be disabled after 90 days of inactivity; this will be a manual review process
+      every 30 days, but the disablement will be automatic.
+      A manual review of all
+      user accounts will be conducted on an annual basis
```

Comment:
Minor: this line wrapping is a bit messy...can we stick with either soft line wrapping (my preference) or hard at 80 columns?